src/openms_gui/source/VISUAL/TOPPViewMenu.cpp

```diff
@@ -175,7 +180,8 @@ namespace OpenMS
     m_help->addAction("Tutorials and documentation", [&]() { GUIHelpers::openURL("html/index.html"); }, Qt::Key_F1);
 
     m_help->addSeparator();
-    m_help->addAction("&About", [&]() {QApplicationTOPP::showAboutDialog(parent, "TOPPView"); });
+    // This is important
```

Comment:
```suggestion
    // It is important to pass parent by value, since the lambda will be evaluated later, even after this function returned and parent reference would be out of scope.
```